CHANGELOG.md

```diff
@@ -16,8 +16,7 @@ Emojis for the following are chosen based on [gitmoji](https://gitmoji.dev/).
 
 ### ✨ New Features
 
-<!-- Scribe data is now loaded into SQLite database tables to make data reference less memory intensive and mitigate crashes. -->
-<!-- Emoji autosuggestions are now available as the user types. -->
+- Added action to command bar information icon
```

Comment:
Thanks for editing the changelog! Could you add a period at the end here and leave the commented features in? I tend to comment them out so I can keep the versions planned 😊